src/common.hpp

```diff
@@ -0,0 +1,124 @@
+/*
+This is a part of Loops project.
+Distributed under Apache 2 license.
+See https://github.com/vpisarev/loops/LICENSE
+*/
+#ifndef __LOOPS_COMMON_HPP__
+#define __LOOPS_COMMON_HPP__
+
+#include "loops/loops.hpp"
+#include <unordered_map>
+#include <stack>
+    
+namespace loops
+{
+    struct syntop
+    {
+        int opcode;
+        std::vector<Arg> args; //TODO(ch): recreate as Arg[10] instead of vector.
```

Comment:
please, don't use std::vector here, use array with some pre-defined (via enum) maximum number of elements